src/main/java/com/snowplowanalytics/snowplow/tracker/emitter/BatchEmitter.java

```diff
@@ -91,6 +93,20 @@ public T bufferCapacity(final int bufferCapacity) {
             return self();
         }
 
+        /**
+         * Provide a denylist of HTTP response codes. Retry will not be attempted if one of these codes
+         * is received. The events in the request will be dropped. An exponential backoff time will
```

Comment:
> The events in the request will be dropped.

I confess I'm not a fan of this solution in the long term. In a general case I don't see the relation between the response code and the tracked events. They could be completely unrelated but we assume they have a relation.
Anyway, this is out of the purpose of this quick enhancement, so it's 👍.